scene/2d/audio_stream_player_2d.cpp

```diff
@@ -296,6 +296,7 @@ float AudioStreamPlayer2D::get_volume_db() const {
 }
 
 void AudioStreamPlayer2D::set_pitch_scale(float p_pitch_scale) {
+	ERR_FAIL_COND(p_pitch_scale <= 0.0);
```

Comment:
I still think that checking `<= 0.0` is risky, as if you're close to 0 it may fail sometimes (when precision errors make you be -0.000001214) and work at other times (when it's 0.000001670).

If we don't want to hardcode a magic number, it should maybe be `< CMP_EPSILON`.